pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/FunctionsBase.java

```diff
@@ -94,14 +183,106 @@ public void registerFunction(final @PathParam("tenant") String tenant,
     })
     @Path("/{tenant}/{namespace}/{functionName}")
     @Consumes(MediaType.MULTIPART_FORM_DATA)
-    public void updateFunction(final @PathParam("tenant") String tenant,
-                               final @PathParam("namespace") String namespace,
-                               final @PathParam("functionName") String functionName,
-                               final @FormDataParam("data") InputStream uploadedInputStream,
-                               final @FormDataParam("data") FormDataContentDisposition fileDetail,
-                               final @FormDataParam("url") String functionPkgUrl,
-                               final @FormDataParam("functionConfig") String functionConfigJson,
-                               final @FormDataParam("updateOptions") UpdateOptions updateOptions) throws IOException {
+    public void updateFunction(
+            @ApiParam(value = "The function's tenant")
```

Comment:
```suggestion
            @ApiParam(value = "The tenant of functions")
```

Reply:
Avoid to use "function's" expression, use "the xxx of xxx", since "function is inanimate".
Check all similar cases.